### DISCUSSION TITLE
Having trouble with OAuth2.0 User Context

### DISCUSSION THREAD

**Discussion by karakotineeraj** [plhery/node-twitter-api-v2#144] [Q&A] [answered]
Can you tell me how I can create a client after I have its **accessToken** and **refreshToken** in OAuth2.0 ?
Maybe there is an answer I couldn't find, so point me in the right direction. 

Thanks in advance.

@alkihis @PLhery @filippkowalski @mika-f

**Comment by alkihis** [comment, marked as answer]
Hi,

An access token, in OAuth 2.0 content, is a **Bearer token**. Just instanciate the class with it.
```ts
const client = new TwitterApi('accessToken')
```

If you want to use your refresh token, instanciate a client with your access token then use `.refreshOAuth2Token` ([see here](https://github.com/PLhery/node-twitter-api-v2/blob/master/doc/auth.md#optional-refresh-the-token-later))

**Reply by karakotineeraj** [reply]
Thanks for the answer.
Just a query: Is this in the docs somewhere explicitly?

**Reply by alkihis** [reply]
In the docs, "Bearer token" is mentionned (in main README). I've added a part to clarify that case, that will be merged on next release.
